### ISSUE TITLE
SpecialK api-ms-win-crt-string-l1-1-0.dll missing

### ISSUE THREAD

**Issue by ghost** [sonic2kk/steamtinkerlaunch#1022]
## System Information
- SteamTinkerLaunch version:  v12.12
- Distribution:  Nobara Linux
- Installation Method:  ProtonUp-QT

## Issue Description
 When trying to launch a game with specialk (in this case mgrr) it complains about having no api-ms-win-crt-string-l1-1-0.dll in the wineprefix despite running on a newer version of specialk. in the wiki section it is said that older specialk versions have this problem, but i tried both default and latest branches and the same error pops up. also, the wiki says there should be 3 branches: stable, nightly and custom, but there is actually default, latest and a blank selection.

**Comment by ghost** [comment]
i'm going to close it as it seems this is fixed on the bleeding-edge version, do consider updating the stable soon though, there's been a lot of commits since march.

**Comment by sonic2kk** [comment]
> do consider updating the stable soon though, there's been a lot of commits since march.

There is already an open issue detailing the plans for v14.0 (it's even pinned, #992), and the reason why it was not originally released: to discourage people from simply defaulting to a "stable", as even when a new stable comes out, you should still use the latest commit you can.

If you want a new stable though, feel free to help out with the items on the roadmap.

*(Duplicate of #889, partially also #893)*

**Comment by NextGenRyo** [comment]
Looking forward to the next release. Using steamtinkerlauncher installed from protonup-qt, version 12.12 and the same issue.

**Comment by sonic2kk** [comment]
Don't use v12.12, please. Just use STL git, it's even available from ProtonUp-Qt, this exact attitude is why I stopped making releases (not just from you). I'm sorry, this is very upsetting and extremely demotivating to read as project maintainer. If people are just going to wait for releases to get features then why am I even making this project. People are just gonna wait until the next release and not bother testing and helping until things are already in a release, defeating the perceived purpose of stability (which I don't abide to, STL by nature is slightly brittle because Steam Client updates can change things alone).

Not to mention, SteamTinkerLaunch is a Bash script. No compilation is even needed, you just need to replace some files, I even spent time with the ProtonUp-Qt maintainer to make this step even easier for people.

There is very, very little reason to ***ever*** use a SteamTinkerLaunch release from the releases section. You should always update as soon as you see a new commit to master. If something regresses, report it and it can get sorted. And before it's merged if it'll take a while, you can install the branch with the fix in the meantime also using ProtonUp-Qt.

It is very insulting, upsetting, and disappointing to see so many people not using the latest git version ☹️ Not just yourself here, just in general. A lot of work goes into this project to find and fix bugs. Why wait on a release that would have the same code changes? A release is simply a tagged commit, zipped up. Take a look in `~/stl/prefix` and compare it to the file structure on GitHub. You could download the files and replace the ones in `~/stl/prefix` with the ones off GitHub. ProtonUp-Qt just downloads and extracts then into that folder. There's nothing special to it, there is no need, ever, to wait on a release to get changes that you want. They're ready for you *right now*! That's what I want to get across.

It's also worth pointing out that when opening issues, you're asked to use the latest git anyway. It only makes sense too, if you encounter an issue, no point in reporting it if it's fixed on master. The natural conclusion to that along with using the latest master being very simple, is to just use it to begin with.

<hr>

My original response was curt and I was a bit grumpy. I apologise for that. I have edited it to take the edge off a little, and provide more context for why I think there is no need to wait for releases.

**Comment by NanoDrivee** [comment]
Just want to add for anyone that stumbles across this issue, you need to enable advanced mode (located rather unintuitively in ProtonUp-Qt's About section) for the option to install SteamTinkerLaunch-git to even show up. Had me confused for a minute.

**Comment by sonic2kk** [comment]
This is documented on SteamTinkerLaunch's [ProtonUp-Qt wiki page](https://github.com/sonic2kk/steamtinkerlaunch/wiki/ProtonUp-Qt#installation).
